examples/nlp/dataflow_diagram.sh

```diff
@@ -0,0 +1 @@
+cat nlp_ops_dataflow.json | dffml dataflow diagram -configloader yaml /dev/stdin
```

Comment:
```suggestion
dffml dataflow diagram nlp_ops_dataflow.json
```

Reply:
Let's avoid the use of `yaml` unless the user is going to look at the contents of the dataflow file, since it requires we install an extra package and slightly complicates things by having them use another command line flag

Reply:
Also, I think that only including the processing stage makes it a bit more clear

Reply:
I uploaded the new version